tabs/pid_tuning.js

```diff
@@ -701,11 +718,8 @@ TABS.pid_tuning.initRatesPreview = function () {
     this.keepRendering = true;
     this.model = new Model($('.rates_preview'), $('.rates_preview canvas'));
 
-    var scale = d3.scale.linear().domain([900, 2100]);
-
-    this.rollScale = scale.range([Math.PI * 2, -Math.PI * 2]);
-    this.pitchScale = scale.range([Math.PI * 2, -Math.PI * 2]);
-    this.yawScale = scale.range([Math.PI * 2, -Math.PI * 2]);
+    this.$superExpo = $('.pid_tuning input[name="show_superexpo_rates"]');
```

Comment:
@mikeller Ok, great. I'm currently in the process of moving layout to Bootstrap fixing as they break. I've picked only grid component for now, but if needed other stuff can be added later. It's stable, modular and has quite a lot of useful things that can be incorporated later.

I guess it would be good to settle on a JS framework if we need one. I personally prefer Backbone.js, but since most of the implement similar patterns I'm happy with other options. Unless it's Angular.js. I guess the most popular right now is React, but it's for views only and comes with proprietary JSX language, which would complicate the process of development. Would be good to hear other people's opinions on this.

As for the coding style — we can take one of the existing like https://github.com/felixge/node-style-guide or http://eslint.org/.

In terms of support for older versions — I'm sure there's some sort of indication whether it's BetaFlight or CleanFlight. I guess @borisbstyle is the one to ask here. Also how old are the versions without BetaFlight tag?